src/components/metadata/index.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import modifiers from '../../utils/modifiers';
+import NodeIcon from '../../components/icons/node-icon';
+import IconButton from '../../components/icon-button';
+import CopyIcon from '../icons/copy';
+import MetaDataRow from './metadata-row';
+import MetaDataValue from './metadata-value';
+import {
+  getVisibleMetaSidebar,
+  getClickedNodeMetaData
+} from '../../selectors/metadata';
+import './styles/metadata.css';
+
+/**
+ * Shows node meta data
+ */
+const MetaData = ({ visible = true, metadata }) => {
+  const [showCopied, setShowCopied] = useState(false);
+
+  const isTaskNode = metadata?.node.type === 'task';
+
+  const runCommandText = !showCopied
+    ? metadata?.runCommand
+    : 'Copied to clipboard.';
```

Comment:
I think that could work, but changing the height would cause the content below to jump down and up which could be a bit jarring? (that said, I still need to add something for that here for multi-line cases)

Reply:
Ah yeah fair. In that case feel free to keep it, but it sounds like you might need to figure out the height issue anyway with some sort of fixed-height message in separate element. I'll ping Gabriel on slack for his thoughts.

Reply:
This has been improved to handle multi-line cases.